CHANGELOG.md

```diff
@@ -27,6 +27,52 @@ The changelog is generated during the release process using [ChangelogLinker](ht
 
 <!-- changelog-linker -->
 
+## [v7.0.0](https://github.com/shopsys/shopsys/compare/v7.0.0-beta6...v7.0.0) - 2019-02-26
+
+### [shopsys/framework]
+
+#### Changed
+
+- [#817] now it is possible to run multiple cron commands
+- [#871] autoload helper classes from Tests\FrameworkBundle\Test namespace
+
+#### Fixed
+
+- [#852] fixed empty cart when no listable product left in cart
+- [#829] fixed js validation constraints
+- [#869] fixed files uploading
+- [#851] flysystem for ImageFacade::copyImages and FileUpload::postFlushEntity is unified
+
+### [shopsys/project-base]
+
+#### Added
+
+- [#825] promocode flow in cart is covered by acceptance tests
+- [#836] implemented picture element
+
+#### Changed
+
+- [#824] Elasticsearch contains products after "build-demo" target now, thanks to [@pejaycz]
+
+### [shopsys/shopsys]
+
+#### Changed
+
+- [#845] shopsys dependencies now using caret carsion
+- [#830] improved documentation "How to set up domains and locales"
+- [#782] changed forms in popup so they work with js validation
```

Comment:
I am really not sure where do some of the entries belong...For example, this particular one seems more like a fix in the framework or a change in the project-base :thinking: However, you do not need to change it now, let us discuss this on some of our meetings in general...